Menus/TitleScreen.cpp

```diff
@@ -111,13 +111,13 @@ namespace RTE {
 		draw_trans_sprite(m_Moon.GetSpriteFrame(0), ContentFile("Base.rte/GUIs/Title/MoonAlpha.png").GetAsBitmap(), 0, 0);
 
 		int starSmallBitmapCount = 4;
-		std::vector<BITMAP *> starSmallBitmaps = ConvertDoublePointerToVectorOfPointers(ContentFile("Base.rte/GUIs/Title/Stars/StarSmall.png").GetAsAnimation(starSmallBitmapCount), starSmallBitmapCount);
```

Comment:
Do we still need the `ConvertDoublePointerToVectorOfPointers` method anymore? May be fine to keep it even if we don't, but I'm curious.

Reply:
Can be useful in case some future:tm: external lib uses this garbage and we can't live without it.